docs/adrs/003-Retry_endpoint_design.md

```diff
@@ -64,8 +64,8 @@ The endpoint takes a job or list of job ids and then attempts to resubmit them t
 #### Q: should the number of retries of a job be limited, and if so, where? e.g. a max_retries field in the parent job? wait and see whether people attempt to rerun jobs that have already failed nine zillion times?
 A: Unknown TBD
 
-#### Q: Preventing the same params from being re-run
-A: We have decided to allow it
+#### Q: Preventing the identical params from being re-run within a retry_jobs request
```

Comment:
Turn this into a question. I'm not sure which of the following interpretations you mean, but either

> How do we prevent the same job from being rerun more than once within a `retry_jobs` request?

or 

> How do we prevent jobs with identical parameters from being rerun more than once within a `retry_jobs` request?

Reply:
done